R/kernel.r

```diff
@@ -219,7 +219,6 @@ shutdown = function(request) {
 
 initialize = function(connection_file) {
     connection_info <<- fromJSON(connection_file)
-    print(connection_info)
```

Comment:
I think we need a debug frag, so that these info can be seen by some parameter...